server/serverConstants.js

```diff
@@ -82,6 +82,8 @@ E.CHROME_EXTENSION_ID = { envVar: 'CHROME_EXTENSION_ID', jsonPath: 'Screensharin
 
 E.ENABLE_ANNOTATIONS = { envVar: 'ENABLE_ANNOTATIONS', jsonPath: 'Screensharing.annotations.enabled', defaultValue: true, parser: parseBool };
 
+E.ENABLE_ROOM_LOCKING = { envVar: 'ENABLE_ROOM_LOCKING', jsonPath: 'enableRoomLocking', defaultValue: true, parser: parseBool };
```

Comment:
Normally I would prefer to make the default match the behaviour of the app before the feature was added but I think this is a reasonable exception. I'm ok with treating this as a new feature that is enabled by default when the code is merged to master branch and made part of any existing instances of the application. If other reviewers disagree, let's discuss.

Reply:
I'm agree with @rhainer, in my opinion the default value should be `false`

Reply:
I'd say have it be false as a default, to match the behavior in the master branch.

Reply:
But after some discussion, I think we've decided that this feature is valuable enough to have it be turned on by default.

Reply:
Yes, let's leave it on by default. I'm choosing to look at this as a security fix that should be encouraged by making it opt-out instead of opt-in.